www/routes/index.tsx

```diff
@@ -76,7 +76,7 @@ function HelloBar() {
       class="bg-gradient-to-r from-blue-200 to-yellow-200 via-green-300 text-black border-b border-green-400 p-4 text-center group"
       href="https://deno.com/blog/fresh-and-vite"
     >
-      Fresh 2 <b>beta release with vite</b>{" "}
+      Fresh 2 <b>release with vite</b>{" "}
```

Comment:
```suggestion
      Fresh 2 <b>released with Vite</b>{" "}
```

IDK, without beta, it just sounds wrong. Probably just being too pedantic tho.

Reply:
For me, it was a bit confusing hearing about 2.0 stable but still seeing the "beta" notification. I prefer consistency, but idk about the devs 🤷‍♂️